app/assets/images/payment_icons/monero.svg

```diff
@@ -0,0 +1,24 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="38px" height="24px" viewBox="0 0 38 24" enable-background="new 0 0 38 24" xml:space="preserve">  <image id="image0" width="38" height="24" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACYAAAAYCAYAAACWTY9zAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```

Comment:
Can you supply a proper SVG instead of a 40x26 base64 encoded PNG?

Reply:
On it.